x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -359,6 +359,10 @@ public final void test() throws Throwable {
                 "CSV tests cannot currently handle CHUNK function",
                 testCase.requiredCapabilities.contains(EsqlCapabilities.Cap.CHUNK_FUNCTION.capabilityName())
             );
+            assumeFalse(
+                "can't use PromQL in csv tests",
```

Comment:
This can be a follow-up but I think it would be good to find a way to add PromQL csv tests, even with PromQL being behind a feature flag.